src/bricks/util/deepCopy.ts

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+export function deepCopy<T>(value: T): T {
+  if (typeof value !== 'object' || value === null) {
```

Comment:
```suggestion
  if (typeof value !== 'object' || value == null) {
```

Aqui vale manter somente dois iguais, para que casos como `0` e `undefined` também se apliquem.

Reply:
Joia, apliquei essa alteração local, executei os testes e eles passaram, obrigado pela sugestão!